docs/manual/en/introduction/Import-via-modules.html

```diff
@@ -62,11 +62,64 @@ <h2>Importing the module</h2>
 		...
 		</code>
 
-		<h2>Caveats</h2>
-
+		<h2>Importable Examples</h2>
 		<p>
-			Currently it's not possible to import the files within the "examples/js" directory in this way.
-			This is due to some of the files relying on global namespace pollution of THREE. For more information see <a href="https://github.com/mrdoob/three.js/issues/9562" target="_blank">Transform `examples/js` to support modules #9562</a>.
+			The core of three.js is focused on the most important components of a 3D engine. Many other components like loaders or controls are part of the
+			examples directory. three.js ensures to keep these files in sync with the core but users have to import them separately if they are required
```

Comment:
> directory. three.js ensures

I'm not sure if there's a precedent set elsewhere in the documentation but `three.js` not being capitalized at the beginning of a sentence here stuck out to me while I was reading it.

Reply:
three.js is not capitalized. :-)

Reply:
Fair enough! I guess it just looked odd because it was at the beginning of a sentence.

Reply:
TBH, I always write it `three.js`^^